hack/prom-rule-ci/rule-spec-dumper.go

```diff
@@ -24,32 +26,55 @@ func main() {
 		os.Exit(1)
 	}
 
-	targetFile := os.Args[1]
+	targetRules := os.Args[1]
+	targetFile := os.Args[2]
 
-	err := rules.SetupRules()
+	var promRule *promv1.PrometheusRule
+
+	if targetRules == "hyperconverged" {
+		promRule = buildHyperconvergedRule()
+	} else if targetRules == "observability" {
+		promRule = buildObservabilityRule()
+	} else {
+		fmt.Printf("ERROR: invalid target rules: %s\n", targetRules)
+		os.Exit(1)
+	}
```

Comment:
nit - can be switch-case, e.g.
```suggestion
	switch targetRules {
	case "hyperconverged":
		promRule = buildHyperconvergedRule()
		
	case  "observability":
		promRule = buildObservabilityRule()

	default:		
		fmt.Printf("ERROR: invalid target rules: %s\n", targetRules)
		os.Exit(1)
	}
```